src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -132,9 +132,21 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
 
         // hide mutex tab
         $(document).ready(()=>{
-            if (genes((window as any).serverVars.theQuery).length <= 1) {
+            if (!(window as any).serverVars.theQuery.trim().length || genes((window as any).serverVars.theQuery).length <= 1) {
                 $('a#mutex-result-tab').parent().hide();
             }
+            //hide gene-specific tabs when no genes are queried
```

Comment:
lets add to this comment to explain why there might be no genes.  namely that we have a geneset query.  we might want to actually have a function isGensetQuery and test against that here, which would remove the ambiguity.